### ISSUE TITLE
Opening a channel in a new tab turns into the new layout.

### ISSUE THREAD

**Issue by AccountMadeJustForThis** [xxzefgh/youtube-classic-extension#42]
Going to a channel without going into a new tab keeps the old layout, however.

**Comment by 0ecd3e** [comment]
I just had this issue, switching the method from "User Agent" to "URL Rewrite" and back seems to be a temporary fix.

**Comment by spodermenpls** [comment]
I've circumvented this by installing https://addons.mozilla.org/de/firefox/addon/disable-polymer-youtube/ on top, since this extension doesn't add the "disable_polymer" line to the URL anymore - it adds double entries to the browser history again, but keeps the Polymer garbage away from channels and the subscription page.

**Comment by MickCasanova** [comment]
Doesn't work for me... I still see the new layout now.

**Comment by 0ecd3e** [comment]
Yeah, I see the new layout too now. Might not be an easy fix anymore.

**Comment by spodermenpls** [comment]
Only "youtube.com" itself makes the new layout kick in on my end, the rest is still sensitive to "disable_polymer" and the Google crawler User Agent, so far.

**Comment by WAZAAAAA0** [comment]
^ yeah the currently working method to get the old YT layout consists of using BOTH:
1. spoofing your UserAgent as GoogleBot
2. adding ?disable_polymer=1 to links as much as possible

I've written a couple of Redirector rules to deal with the second point, instructions here: https://old.reddit.com/r/youtube/comments/hoijz2/restoring_the_old_yt_prepolymer_layout_as_of/fxjyyqe/
